naughty/fedora-38/4231-iscsi-selinux

```diff
@@ -0,0 +1 @@
+error: Failed to start pool iscsi-pool
```

Comment:
There's not enough context here IMHO. Could this grow a test case name/traceback context, if we don't get an actual error message in the test output? E.g. this could easily happen if we deliberately test adding an iSCSI  pool with invalid parameters.

Reply:
Indeed. ok let me repost

Reply:
Verified with logs from https://artifacts.dev.testing-farm.io/d66356c1-cdc2-4b3b-9b1a-d482967799f7/